### PR TITLE
Update Ubuntu/Mint install instructions

### PR DESCRIPTION
**Pull request by martinpitt**
See individual commits for details.

Preview: https://martinpitt.github.io/cockpit-project.github.io/running.html#ubuntu

### CODE REVIEW

File: running.md [hunk 1]
```diff
@@ -231,7 +231,7 @@ When updating Cockpit-related packages and any dependencies, make sure to use `-
 {:.note}
 These commands require a POSIX compatible shell like `bash`. For other shells like `fish`, temporarily run `bash -i`.
 
-Cockpit is available in Ubuntu 17.04 and later, and available [as an official backport](https://help.ubuntu.com/community/UbuntuBackports) for 16.04 LTS and later.
+Cockpit is available in all current Ubuntu releases, and available [as an official backport](https://help.ubuntu.com/community/UbuntuBackports) for LTS releases.
```
Comment (allisonkarlitskaya) [inline running.md:234]: 16.04 is surely the last supported LTS release as well, no?  Could we say "all current Ubuntu releases, including all currently supported LTS releases"?

**Review by allisonkarlitskaya — APPROVED** [review]
Quite fine with this.  One question.

File: running.md (hunk 1 above)
Comment (garrett) [reply running.md:234]: "Supported" could be interpreted as our own support, so I think we should avoid that, unless we find a clear way to say it. I agree that calling out LTS here would be redundant... and "current" would include LTS, IMO.

We could say "maintained" instead, or we could simplify it completely.

```suggestion
Cockpit is available in maintained Ubuntu releases, and available [as an official backport](https://help.ubuntu.com/community/UbuntuBackports) for LTS releases.
```

```suggestion
Cockpit is available in Ubuntu, with [updated versions in official backports for LTS releases](https://help.ubuntu.com/community/UbuntuBackports).
```

(I prefer something like this second suggestion. WDYT?)

Comment (martinpitt) [reply running.md:234]: I actually just thought "let's just say Ubuntu". So fully agreed, I'll take the second suggestion. Thanks!

**Pull request merged**